### DISCUSSION TITLE
Sum of size of files, getting different results with du

### DISCUSSION THREAD

**Discussion by rieje** [sharkdp/fd#1499] [Q&A] [answered]
I am looking to find the size of a directory of the filesystem (ignoring bind mounts). I want to use `fd` to show sorted list of the largest files. In finding the command for this, I found the following two commands yields different sum of files of the same directory:

* `fd -0 --hidden --one-file-system -t f . ~/.dir | du --files0-from=- -hc`
* `du -xhs ~/.dir`

Where do `fd` and `du` differ in these 2 commands? The first reports 5.8G for me and the second reports 7.3G.

Any ideas much appreciated.

**Comment by tmccombs** [comment, marked as answer]
Do you have files exluded by .gitignore? Do you get the same answer if you also pass `--no-ignore`?

**Reply by rieje** [reply]
That was it, thank you!
